### DISCUSSION TITLE
[1286280] 弃海[仅本体]

### DISCUSSION THREAD

**Discussion by 1271620983** [pjy612/SteamManifestCache#420] [Share]
`appid:1286280`
keys:
```
1286281;ebb916a2868eb6ee88612d8d67b81fcc0465b2826adcc9dbbab243f6ece24541
```
[[1286280]弃海.zip](https://github.com/pjy612/SteamManifestCache/files/14033573/1286280.zip)

code:39A2D11F7B59B3A4
之前的内测码写错了:(

**Comment by pjy612** [comment]
现在不用 code，看更新说明即可
